src/components/UI/Typography/Typography.tsx

```diff
@@ -15,22 +15,29 @@ interface SharedTypographyProps extends Omit<TextProps, 'variant' | 'style' | 's
   children?: React.ReactNode;
   style?: CSSProperties;
   size?: CustomSize;
+  'data-testid'?: string; 
```

Comment:
data test id should be static and not prop because we don't want this to change otherwise some tests will get broken after we will change the name of the datatest id